src/components/ControlLink/ControlLink.scss

```diff
@@ -0,0 +1,41 @@
+@import '~om';
+
+.ControlLink {
+  &:not(&--disabled) {
+    cursor: pointer;
+  }
+  width: fit-content;
```

Comment:
I don't think this is required.

Reply:
I think I have seen it grow to 100% width but not sure. I can remove.

Reply:
Make it `display: inline-flex`.